### DISCUSSION TITLE
Go pro labs won’t save for each boot up

### DISCUSSION THREAD

**Discussion by Apexviii** [gopro/labs#992] [General]
I can’t get the qr settings to boot up with every battery change. 

My Old go pro would and I can’t figure out what I’m doing wrong on this hero11.  

The old GoPro was a hero 9

**Comment by dnewman-gpsw** [comment]
I expect you are not setting up a BOOT command.  Please share the setting you are expecting to survive a reboot.

**Reply by Apexviii** [reply]
I’m using the QR control ap. If I shared the QR code would you be able to see all my settings from there?  Or what in the qr control app would make it change to every reboot ?

**Reply by dnewman-gpsw** [reply]
Yes the QR code will help, but the command string displayed below it is more useful. The QRControl app has no support for BOOT commands, other than those that are manually entered. Only settings that are permanent, are the same as a stock camera.  If you set a resolution or an ISO, that should be remembered (with a normal shutdown.) If you use Labs extensions like BITR, that will be remembered if you set the "Permanent" flag.  Nothing else is remembered by design (even on your older HERO9.)

**Reply by Apexviii** [reply]
Hmm interesting. Cuz on the old one it
Would keep the speed activation settings even when I removed the battery. And I only used the qr app or the go pro qr webpage 

Idk how I did it tho but it was doing that 
![QR-mVr4p60fWe3oSMaAhS0dR0wAsLcGb1d1i](https://github.com/user-attachments/assets/1cec67d3-4b20-4126-9fd6-63079f805189)

**Reply by dnewman-gpsw** [reply]
It shouldn't have been without a boot command, and I'm 100% sure (as the lead developer) it wasn't. 
The above is a not optimal command. Don't combine settings with actions.

Add all the settings in one, in it own command.  So the ```mVr4p60...``` section in a QR Code.

The Speed trigger, add to a separate QR code.  While they can be combined, the issue is you are repeating.  This repeat is repeating everything, including all your settings, slowing everything down.  

Speed trigger should be only ```!SK16-13!R``` 

If you want this to be automatic on boot, you need it to explicitly request that like:   
```*BOOT="!Lbt"!SAVEbt=!SK16-13!R```

**Reply by Apexviii** [reply]
I’m not gunna lie. I’m super new to all of this so all I know is how to do the QR code scanning. 

I use the go pro for wake surf videos, so all I really need outside of what the camera can do,
Is just auto record at16kph and stop under 13


I’ll loook on YouTube for videos on how to do it like you guys are saying with the text but like I said I know nothing about that    So I gotta learn

But I do want it to keep those settings on boot.

**Comment by Apexviii** [comment]
I think see how to do it,  I cleared all the current settings and added the text. So I just need to scan this and it will be like I want.  I’ll test in a few hours when I’m home 
![Uploading IMG_5961.png…]()
